localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -961,10 +961,10 @@ msgid "points"
 msgstr "puntos"
 
 msgid "Revert gap between characters"
-msgstr "Revert gap between characters"
+msgstr "Revertir el espacio entre caracteres"
 
 msgid "Distance between characters"
-msgstr "Distancia entre caracteres"
+msgstr "Distancia entre carácteres"
```

Comment:
The oficial word from the RAE is caracteres and not carácteres. Please check: https://www.rae.es/desen/car%C3%A1cter

Reply:
Correct. Changed.